lib/hyrax/migrator/crosswalk_metadata_preflight.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal:true
+
+require 'hyrax/migrator/crosswalk_metadata'
+require 'byebug'
```

Comment:
I think we should remove this line for byebug so that it's ready for OD1 production. In dev this task works great! In prod env, we would get:
```
rake aborted!
LoadError: cannot load such file -- byebug
```